pkgs/by-name/ar/ares-cli/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, mkYarnPackage
+, fetchFromGitHub
+}:
+mkYarnPackage rec {
+  pname = "ares-cli";
+  version = "3.0.6";
+  src = fetchFromGitHub {
+    owner = "webos-tools";
+    repo = "cli";
+    rev = "v${version}";
+    hash = "sha256-HhS49jg6PrWzXEX5P+NgcznMZw4yJyI0EYoCZ8y6UeU=";
+  };
+
+  yarnLock = ./yarn.lock;
+  doDist = false;
```

Comment:
```suggestion
  postPatch = ''
    ln -s npm-shrinkwrap.json package-lock.json
  '';

  dontNpmBuild = true;
  npmDepsHash = "sha256-fTEYQY0ZhwYKnc7+EsgnnGovYJ8w+norq7oLnVrWNZg=";
```

Reply:
`buildNpmPackage` works nicely with `npm-shrinkwrap`, just symlink it to `package-lock.json`.